vcf2fhir/fhir_helper.py

```diff
@@ -111,9 +132,11 @@ def add_regionstudied_obv(
             self,
             ref_seq,
             reportable_query_regions,
-            nocall_regions):
-        if reportable_query_regions.empty and nocall_regions.empty:
-            return
+            nocall_regions,):
+        if not (reportable_query_regions is not None and
+                len(reportable_query_regions) == 0):
+            if (reportable_query_regions.empty and nocall_regions.empty):
+                return

```

Comment:
Can we combine all the conditions in both if ?
```suggestion
        if not (reportable_query_regions is not None and
                len(reportable_query_regions) == 0) and
             reportable_query_regions.empty and nocall_regions.empty:
                return

```